spec/views/casa_cases/show.html.erb_spec.rb

```diff
@@ -11,22 +11,22 @@
   end
 
   context "when there is court date" do
-    let!(:casa_case) { create(:casa_case, :with_upcoming_court_date, casa_org: organization, case_number: "111") }
-    let(:date) { casa_case.court_dates.map(&:date).first.to_date.strftime("%B %d, %Y") }
+    it "renders casa case with court dates" do
+      casa_case = create(:casa_case, case_number: "111")
+      create(:court_date, casa_case: casa_case, date: Date.new(2023, 5, 6))
 
-    before { assign(:casa_case, casa_case) }
-    it "render casa case with court dates" do
+      assign(:casa_case, casa_case)
       render
 
-      expect(rendered).to match(casa_case.case_number)
-      expect(rendered).to match(date)
+      expect(rendered).to match("111")
+      expect(rendered).to match("May 6, 2023")
     end
 
-    it "render button to add court date" do
-      render
+    # it "render button to add court date" do
+    #   render
 
-      expect(rendered).to have_content("Add a court date")
-    end
+    #   expect(rendered).to have_content("Add a court date")
+    # end
```

Comment:
@DariBerrie any reason why this is commented out?

Reply:
Oh, can you just add `casa_case = create(:casa_case)` into the spec? It's missing now since you removed the `let` statement.

Reply:
You can add it right before the `render` statement on line 26.

Reply:
Oops, looks like this got merged before you could fix it. I made the change in #4802 .